### PR TITLE
Add query for regional breakdown

### PR DESCRIPTION
**Pull request by priyesh777**
## Changes:
- Add query changes to regional breakdown
- Add tooltip for pie chart component

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] codegen errors
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

### CODE REVIEW

File: app/views/Dashboard/Overview/PercentageCardGroup/index.tsx [hunk 1]
```diff
@@ -10,49 +10,211 @@ import {
...
+            region: $region,
+        }
+        ) {
+            id
+            contextIndicatorValue
+            contextDate
+            emergency
+        }
+    }
+`;
```
Comment (subinasr) [inline app/views/Dashboard/Overview/PercentageCardGroup/index.tsx]: Why are we calling the same queries twice? Can we merge this into a same query?

File: app/views/Dashboard/Overview/PercentageCardGroup/index.tsx [hunk 2]
```diff
@@ -10,49 +10,211 @@ import {
...
+        ) {
+            id
+            contextIndicatorValue
+            mostRecent
+            emergency
+            contextDate
+            region
+        }
+    }
+`;
```
Comment (subinasr) [inline app/views/Dashboard/Overview/PercentageCardGroup/index.tsx]: This too!

**Review by subinasr — CHANGES_REQUESTED** [review]

File: app/views/Dashboard/Country/index.tsx [hunk 3]
```diff
@@ -125,6 +126,7 @@ const COUNTRY_PROFILE = gql`
                 iso3: $iso3,
                 contextIndicatorId:$contextIndicatorId,
                 emergency: $emergency,
+                isTwelveMonth: $isTwelveMonth,
```
Comment (AdityaKhatri) [inline app/views/Dashboard/Country/index.tsx:135]: if always true, directly set true.

File: app/views/Dashboard/Overview/PercentageCardGroup/index.tsx [hunk 4]
```diff
@@ -10,49 +10,211 @@ import {
...
+} from '#generated/types';
 
 import styles from './styles.css';
+import { FilterType } from '../../Filters';
 
 interface PercentageCardGroupProps {
     className?: string;
+    filterValues?: FilterType | undefined;
     uncertaintyChartActive: boolean;
 }
```
Comment (AdityaKhatri) [inline app/views/Dashboard/Overview/PercentageCardGroup/index.tsx:53]: Move props closer to function

File: app/views/Dashboard/Overview/PercentageCardGroup/index.tsx [hunk 5]
```diff
@@ -10,49 +10,211 @@ import {
...
+            isGlobal: $isGlobal,
+            mostRecent: $mostRecent,
+            region: $region
+        }
+        ) {
+            contextIndicatorValue
+            emergency
+        }
+    }
+
```
Comment (AdityaKhatri) [inline app/views/Dashboard/Overview/PercentageCardGroup/index.tsx]: remove empty line

File: app/views/Dashboard/Overview/PercentageCardGroup/index.tsx [hunk 6]
```diff
@@ -10,49 +10,211 @@ import {
...
+    $region: String,
+    ) {
+        epiDataGlobal(
+        filters: {
+            isTwelveMonth: $isTwelveMonth,
+            isGlobal: $isGlobal,
+            emergency: $emergency,
+            contextIndicatorId: $contextIndicatorId,
+            region: $region,
+        }
```
Comment (AdityaKhatri) [inline app/views/Dashboard/Overview/PercentageCardGroup/index.tsx:128]: fix indent plz

File: app/views/Dashboard/Overview/PercentageCardGroup/index.tsx [hunk 7]
```diff
@@ -10,49 +10,211 @@ import {
...
+                [outbreak.emergency]: outbreak.contextIndicatorValue,
+            }
+        ))
+    ), [outbreakResponse?.epiDataGlobal]);
+
+    const totalCase = totalOutbreakCasesResponse?.epiDataGlobal
+        .find(
+            (emergency) => emergency.emergency === filterValues?.outbreak,
+        );
+
```
Comment (AdityaKhatri) [inline app/views/Dashboard/Overview/PercentageCardGroup/index.tsx]: use useCallback

File: app/views/Dashboard/Overview/PercentageCardGroup/styles.css [hunk 8]
```diff
@@ -10,14 +10,14 @@
     }
 
     .trends-card {
-        flex-basis: calc(42% - var(--dui-spacing-small));
+        flex-basis: calc(35% - var(--dui-spacing-small));
```
Comment (AdityaKhatri) [inline app/views/Dashboard/Overview/PercentageCardGroup/styles.css]: sir maths

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

**Review by shreeyash07 — APPROVED** [review]

File: app/views/Dashboard/Country/index.tsx (hunk 3 above)
Comment (shreeyash07) [reply app/views/Dashboard/Country/index.tsx:135]: It may vary in some cases in future

**Review by subinasr — APPROVED** [review]

**Pull request merged**